setup.py

```diff
@@ -81,7 +81,9 @@
         # dependency of contextvars, which we vendor
         "immutables==0.6",
     ],
-    extras_require={"latex": ["ipython"]},
+    extras_require={"latex": ["ipython"],
+                    "tutorials": ["forest-benchmarking", "jupyter", "matplotlib", "seaborn",
+                                  "pandas", "scipy", "tqdm"]},
```

Comment:
What's up with this change? Why do we need to list these dependencies if we're deleting all the examples from this repo?

Reply:
This is going to be added to the forest-tutorials README:

Running the notebooks without using Docker
------------------------------------------

We recommend using the Binder link above, or following the Docker-based instructions, if all
you want to do is run the notebooks. This is because it requires no setup on your end (except
for maybe installing Docker). However, if you'd like to run the notebooks locally without Docker,
or if you'd like to make a pull request and contribute your own tutorial, you'll have to set up
a Python environment with the necessary requirements installed. Normally, these are defined in a `requirements.txt` file, but adding one to this repo would cause Binder to use that for its
configuration instead of the `Dockerfile`. Thus, we provide the requirements for the tutorials
as a convenient `extras_require` extension to pyQuil's setup.py file. Thus, run the following command in your Python environment to install all the necessary requirements:

```bash
pip install "pyquil[tutorials]"
```

Then, assuming you've cloned this repository and you're in its top-level directory, running
`jupyter notebook` should be all you need to get started!

Reply:
Make sense. Maybe another option would be to have the requirements live in the tutorials repo, but with a nonstandard file name, like `tutorial-requirements.txt`, then `pip install -r tutorial-requirements.txt`.

But this seems fine, too (especially because the `tutorial-requirements.txt` thing might not work).

Reply:
that's also a good idea -- but there's another caveat that im trying to work around: i want to install all these requirements during the build process of the forest-notebook repo (different from forest-tutorials) which means that the tutorial-requirements file would have to live there. that would be a reasonable second option, but i feel like people are less likely to clone that repo than pyquil idk

Reply:
happy to do either tho, lmk what you think

Reply:
I guess if it has to live somewhere other than the `forest-tutorials` repo, pyquil probably makes more sense than `forest-notebook`.